cmd/kubeadm/app/cmd/config.go

```diff
@@ -45,10 +45,18 @@ import (
 const (
 	masterConfig = "MasterConfiguration"
 	nodeConfig   = "NodeConfiguration"
-	sillyToken   = "abcdef.0123456789abcdef"
 )
 
-var availableAPIObjects = []string{masterConfig, nodeConfig}
+var (
+	availableAPIObjects = []string{masterConfig, nodeConfig}
+	// sillyToken is only set statically to make kubeadm not randomize the token on every run
+	sillyToken = kubeadmapiv1alpha2.BootstrapToken{
```

Comment:
The question I had earlier is do we want to tie this object to a specific api-version, b/c it shouldn't mutate.